src/pages/core/external-plugins/app-data.md

```diff
@@ -0,0 +1,250 @@
+---
+title: AppData Plugin
+metaTitle: Core - AppData Plugin
+description: Learn about the MPL Core AppData Plugin
+---
+
+## What is an AppData Plugin?
+
+The `AppData` third party plugin stores and contains arbitrary data that can be written to by the `dataAuthority`. Note this is different then the overall plugin authority stored in the `ExternalRegistryRecord` as it cannot update/revoke authority or change other metadata for the plugin.
+
+Think of AppData as like a partitiion data area of an Asset that only a certain authority can change and write too.
+
+This is usefull for 3rd party sites/apps to store data they made need to execute certain functionality within their product/app.
+
+## Works With
+
+|                       |     |
+| --------------------- | --- |
+| MPL Core Asset        | ✅  |
+| MPL Core Collection\* | ✅  |
+
+\* MPL Core Collections work with the LinkAppData Plugin.
```

Comment:
"can also work" I think implies that you can use either AppData or LinkedAppData, which I think makes sense. But maybe we should list them separately?